spec/models/service_template_ansible_tower_spec.rb

```diff
@@ -25,31 +25,83 @@
       }
     end
 
-    it 'creates and returns an ansible tower catalog item' do
-      service_template = ServiceTemplateAnsibleTower.create_catalog_item(catalog_item_options)
-      service_template.reload
+    context '.create_catalog_item' do
```

Comment:
Although context and describe seem interchangeable, my understanding the typical use is 
describe a method
context a condition